src/codegen.cpp

```diff
@@ -5964,7 +5961,7 @@ static void init_julia_llvm_env(Module *m)
         &pow,
```

Comment:
My main concern would be whether those are the same function. I would expect that the C++ version is getting name mangled might not end up picking up the openlibm version.